README.md

```diff
@@ -46,27 +114,71 @@ Defining them is part of the set up of your project.
-This script is called `./scripts/genesisEnvironmentVariables.ts`.
-You can execute it by running `yarn environment`.
+### How do I get mine?
+
+You have three different options:
 
-Setting up is a step by step process.
-You will need to run the script repeatedly and follow the instructions that it provides, depending on your project's state.
-After running this script each time, you need to manually copy the output and save it on the .env-file on the main project folder.
+1. Using `yarn environment`:
 
-## Process
+   There is a script to facilitate the generation of the environment variables inside of this repository.
+   This script is called `./scripts/genesisEnvironmentVariables.ts`.
+
+   - You can execute it by running `yarn environment`.
+
+   Setting up your environment with this script is a step by step process.
+   You will need to run the script repeatedly and follow the instructions that it provides, depending on your project's state.
+   After running this script each time, you need to manually copy the output and save it on the `.env`-file .
+
+2. Using the [kilt-distillery-cli](https://github.com/KILTprotocol/kilt-distillery-cli)
+
+   This is a _Command Line Interface_ tool that can help you obtaining this variables and also does other common tasks (unrelated to this project).
+   The distillery uses the same key derivation as this repository, which means that it is highly compatible.
+
+3. Without help:
+
+   If you are a _pro_, you could defined and generate them externally and add them to the `.env`-file.
+   You would probably have to modify the `generateKeyPairs.ts` files (on _scripts_ and _backend_) to match your key derivation though.
+
+## Well-Known-DID-Configuration
+
+After having all your environment variables, to build a **Domain Linkage Credential**:
+
+- run: `yarn well-known-config`.
+
+If you want know more about this, check out the [Identity Foundation Documentation](https://identity.foundation/.well-known/resources/did-configuration/).
+[Our dApp documentation also have a section about this.](https://docs.kilt.io/docs/develop/dApp/well-known-did-config)
+
+## Starting the dApp locally
+
+After having all your environment variables and your well-known-did-configuration:
+
+- run `yarn start`.
+
+## Login Process
+
+After having set up the whole project, when the website is up and running, the login process can take place.
+Each user that wants to login would trigger the process that is displayed below.
+The process is full of HTTPS queries and extension-api's messages.
+
+A user could trigger the whole process with just one click on the website, for example "Login with extension X" or "Login with extension Y".
+
+On this example project, several (2-3) interactions are need because it is broken down into little steps.
+We encourage you to open and read the browser's and backend's consoles.
+Try to understand what is happening on each step, which function is responsible for what, how did the cookies changed.
```

Comment:
This sounds like it's a Uni assignment. 🎓 Or maybe a challenge. They shouldn't `try to understand`. They should simply read and understand

Reply:
You would rather write something like `This will help you to understand`.

Reply:
It's `This should help you to understand` now.

Reply:
It's `This should help you understand` now.**   (grammar)